### ISSUE TITLE
Login page Database design 

### ISSUE THREAD

**Issue by lingibelli** [atthelocal/atthelocal#9]
As a web developer, I need database table for user. The database table should have the below fields.

Table Name : Users

Fields:
Id - Number Primary Key
username - unique
password - hash key encription
emailId 
phone
joinDate  - timestamp
modifiedDate - timestamp
primaryLocation - string
profileSettings  - string - Json

**Comment by saran-mani** [comment]
which type of data contain this profileSettings - string - Json?

**Comment by lingibelli** [comment]
> which type of data contain this profileSettings - string - Json?
 
@saran-mani , I was thinking to build a long JSON string with all the user settings and save it in this varchar/blob column in the database. What is your thought?

@nurav05 , Ramesh maam, What is your thought?

**Comment by saran-mani** [comment]
> > which type of data contain this profileSettings - string - Json?
>  
> @saran-mani , I was thinking to build a long JSON string with all the user settings and save it in this varchar/blob column in the database. What is your thought?
> 
> @nurav05 , Ramesh maam, What is your thought?

ok i understood. 

I attached a sample database schema

![IMG-20240207-WA0003.jpg](https://github.com/atthelocal/atthelocal/assets/57593599/96d0f21c-281c-4201-a0f2-5135c8cf5770)

**Comment by lingibelli** [comment]
@saran-mani , Thank you Saran. Sounds good. Would you mind creating this as a SQL file. We can add the sql file to github repository and append the file with new tables as we create.  Thank you.

Also does the snowflake symbol against username ,email and phone means unique values?

**Comment by saran-mani** [comment]
@lingibelli yes you're correct the snowflake represents the data is unique Values.
